charts/boundary-chart/values.yaml

```diff
@@ -49,6 +49,17 @@ gcpckms:
   key_ring: boundary-keyring
   crypto_key: boundary-key
 
+azurekeyvault:
+  enabled: false
+  # azurekeyvault Documented Here:
+  # https://developer.hashicorp.com/boundary/docs/configuration/kms/azurekeyvault 
+  purpose: root
```

Comment:
@janikgar nit: not needed since we are in the cm templates, we are populating the list (root, recovery and worker-auth). This also applies to the other kms options in the helm values file.